lib/common/schemas.c

```diff
@@ -96,32 +95,20 @@ pcmk__find_x_0_schema_index(GList *schemas)
         return best;
     }
 
-    CRM_ASSERT(schemas != NULL);
```

Comment:
You could leave the CRM_ASSERT on known_schemas, which would also enforce the assumption in `xml_latest_schema_index` that `crm_schema_init` was called first.